src/test/java/com/powsybl/openloadflow/OperationalLimitsTest.java

```diff
@@ -47,7 +48,7 @@ void testLineCurrentLimits() {
         Network network = EurostagTutorialExample1Factory.createWithFixedCurrentLimits();
         List<LfNetwork> lfNetworks = LfNetwork.load(network, new MostMeshedSlackBusSelector());
         assertEquals(1, lfNetworks.size());
-        LfNetwork lfNetwork = lfNetworks.get(0);
+        LfNetwork lfNetwork = lfNetworks.stream().filter(n -> n.getNumCC() == ComponentConstants.MAIN_NUM && n.getNumSC() == ComponentConstants.MAIN_NUM).findAny().orElseThrow();
```

Comment:
These is boiler plate code. We have it everywhere in unit tests. Either we move it to a utility class of better we can just rely on the fact that first network is always the (main, main)  one.